QUEST_LV_0100.tsv

```diff
@@ -1440,9 +1440,9 @@ QUEST_LV_0100_20150317_001439	We are still reviving the city, but since it's too
 QUEST_LV_0100_20150317_001440	Follower Alvidas
 QUEST_LV_0100_20150317_001441	Since follow believe the Goddess Saule, you'll be able to accept this as a high difficulty such. {nl} impiety to, do you think want to who stay like this terrible place?
 QUEST_LV_0100_20150317_001442	Follower Laminta
-QUEST_LV_0100_20150317_001443	$If that vicious atmosphere spreads to other forests..{nl}Those forests will also change, like Kvailas Forest.{nl}That this could happen to the whole kingdom.. it scares me..
+QUEST_LV_0100_20150317_001443	$If that evil aura spreads to other forests..{nl}Those forests will also change, like Kvailas Forest.{nl}That this could happen to the whole kingdom... It scares me...
```

Comment:
It sort of works. But grammar wise, I'm hoping we can make a fuller sentence. 

How about adding "then" after "and" a "may also " at the end
And then the entire Kingdom may also...